.github/.linkspector.yml

```diff
@@ -4,11 +4,5 @@ useGitIgnore: true
 excludedFiles:
   - SECURITY.md # Ignore this as the aka links see to keep breaking
 ignorePatterns:
-  # Ignore links to the GitHub repository itself as this will fail until its public. This is mostly used by the Go autogen docs.
```

Comment:
Are these changes intended?

Reply:
most probably the feature branch needs to be updated - i will check if this coming from my branch